spec/discourse_api/api/categories_spec.rb

```diff
@@ -85,4 +85,27 @@
     end
   end
 
+  describe "#category_set_user_notification" do
+    before do
+      stub_post("#{host}/category/1/notifications").to_return(body: fixture("notification_success.json"), headers: { content_type: "application/json" })
+    end
+
+    it "makes the post request" do
+      response = subject.category_set_user_notification(id: 1, notification_level: 3)
```

Comment:
Should just be `1`?

Reply:
I don't believe so. I backfilled this test for the existing `category_set_user_notification` method. The new method is `category_set_user_notification_level`. So, this is the existing syntax.

https://github.com/discourse/discourse_api/blob/99446adaa476d11f058552440199f47d962d66f8/lib/discourse_api/api/categories.rb#L78-L83

Reply:
Oh yes, you are right! Forgot about the deprecated method for a sec.